packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -4505,9 +4505,11 @@ export default class Meeting extends StatelessWebexPlugin {
     options: {
       joinOptions?: any;
       mediaOptions?: AddMediaOptions;
+      isRetry?: boolean;
+      prevJoinResponse?: any;
```

Comment:
Carrying over first successful join response to retry attempt because otherwise if you were the first person to join/start a meeting the meeting would be teared down before trying to join again